doc/reference/plutus/plutus-minting-script-example.md

```diff
@@ -0,0 +1,85 @@
+# Plutus Minting Scripts
+
+## What is a Plutus minting script?
+
+This is a type of Plutus script that is required to validate the minting of multi-asset tokens. We can do this using Mary-era scripts. However, Plutus scripts allow us to encode more logic beyond requiring verification keys and timelocks. The Plutus minting script expects only a redeemer in order to successfully validate the minting of a multi-asset.
```

Comment:
Suggest slight rephrasing for better clarity:
'A Plutus minting script is required to validate the minting of multi-asset tokens. Unlike Mary-era scripts, Plutus scripts support more logic without requiring verification keys and timelocks. The Plutus minting script expects *only* a redeemer in order to successfully validate the minting of a multi-asset.'